.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    # Set environment variables that depend on other environment variables.
+    - run: |
+        echo ::set-env name=WIT_PACKAGE_UNDER_TEST_PATH::"$GITHUB_WORKSPACE/wit-packages/api-scala-sifive.git"
+        echo ::set-env name=WIT_WORKSPACE::"$GITHUB_WORKSPACE/wit-workspace"
+
+    - uses: actions/checkout@v2
+      with:
+        path: ${{ env.WIT_PACKAGE_UNDER_TEST_PATH }}
+
+    - name: Wit init
+      run: |
+        mkdir -p "$WIT_WORKSPACE"
+        docker run \
+          --volume="$WIT_PACKAGE_UNDER_TEST_PATH:/mnt/api-scala-sifive.git" \
+          --volume="$WIT_WORKSPACE:/mnt/workspace" \
+          --workdir="/mnt/workspace" \
+          --rm \
+          sifive/wit:v0.11.1 \
```

Comment:
Do we have a way to query which version the latest release is?
May help reduce some of the churn on version bumps

Reply:
Docker image tags are just strings, and unlike Git, where you are generally discouraged from overwriting tags, it seems like the common convention for Docker images is to overwrite certain tags when newer images are created. For example, every time we create a new point release of `0.11.x`, we could overwrite the `0.11` tag to also point to it.

Since I don't think we're at the point where we can maintain backwards compatibility indefinitely, I don't think we should always point to the latest release, but I think it's fine for us to let the patch version float.

In any case, we don't really have the infrastructure set up at the moment to manage Docker image tags like this, but it's probably something we should set up soon.